B/BVPInterface/build_tarballs.jl

```diff
@@ -0,0 +1,119 @@
+using BinaryBuilder
+
+name = "BVPInterface"
+version = v"0.0.1"
```

Comment:
IF you have to make up a version number, for your own sanity I'd suggest using something like 0.1.0 or 1.0.0.  All versions in the 0.0.x are considered breaking, which is a pain to deal with